### ISSUE TITLE
sphinx doesn't like footnotes that are not referenced in the text

### ISSUE THREAD

**Issue by llimeht** [SasView/sasmodels#378]
In https://github.com/SasView/sasview/pull/1401 we considered turning warnings emitted by sphinx into errors; that would certainly be a good plan to find and fix syntax errors within the documentation (I've started down this rabbit hole already). 

The current approach to providing citations within the `sasmodels/models/*.py` is to include the details of the relevant publication in a footnote. Sometimes the footnotes are just included at the bottom of the documentation, and sometimes they are given explicit labels and referenced from the text. Unfortunately, footnotes that are included in a page but not referenced at all within the page generate warnings:

```
sasmodels/doc/model/hollow_cylinder.rst:97: WARNING: Footnote [#] is not referenced.
```

Basically, sphinx wants either a "reading list" with no in-text reference to the citation or a "references" section where there are no citations that are missing an in-text reference. The current documentation sits awkwardly between the two. 

There are two possible approaches here:
 * remove all in-text citations and turn the list of papers into a simple numbered list
 * alter the text to include a reference to all footnotes

The latter seems likely to be more useful, but we might end up throwing a sentence at the bottom of the documentation that is there only to be a placeholder for references, such as "For further information, see [1] and [2]." Does that sound OK?

**Comment by butlerpd** [comment]
Sounds ok to me.  Not too onerous moving forward I think, but guess that going through all the models and adding such a sentence and changing any docs that say how to write the docs (are there any?) might be a bit of work.
